lib/rails/observers/railtie.rb

```diff
@@ -35,7 +35,7 @@ class Railtie < ::Rails::Railtie
           # which eventually calls `observed_class` thus constantizing `"User"`,
           # the class we're loading. 💣💥
           require "active_record/base"
-        rescue LoadError
+        rescue LoadError, NameError
```

Comment:
instead of doing this we should be requiring `active_record` before `active_record/base`. Can you change it?

Reply:
@rafaelfranca Yup, makes sense. Updated!